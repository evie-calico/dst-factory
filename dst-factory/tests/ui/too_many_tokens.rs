use dst_factory::make_dst_factory;

#[make_dst_factory(no_std, XXX)]
struct ToManyTokens {
    id: i32,
    data: str,
}

fn main() {
}
