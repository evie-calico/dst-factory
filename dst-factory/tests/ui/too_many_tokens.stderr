error: Unexpected input
 --> tests/ui/too_many_tokens.rs:3:26
  |
3 | #[make_dst_factory(no_std, XXX)]
  |                          ^
